src/features/Novel/components/NovelTabs/NovelTabs.module.css

```diff
@@ -0,0 +1,40 @@
+.list {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  list-style: none;
+  border-bottom: 1px solid var(--chakra-colors-gray-300);
+}
+
+.list li {
+  display: flex;
+}
+
+.list li:not(:last-child) {
+  margin-right: 24px;
+}
```

Comment:
Using tags as selectors isn't best choice. It is recommended to use classes over tags.

Reply:
Using tags and classes is inconsistent ☝️. Also can lead to bugs when refactoring🐞.